### DISCUSSION TITLE
(Recovered, May 16, 2021) Temporarily failed in Surge

### DISCUSSION THREAD

**Discussion by fu-sen** [sintaxi/surge#451] [General]
It looks like there was a temporary problem with some Surge environments.
`www.surge.sh` could not be referenced, and some CDN edge servers also failed to resolve names.

If you come across it, you can work around it by pointing your name server to a specific CDN edge server
`SUBDOMAIN.surge.sh`. or `SUBDOMAIN.surge.world`

https://gist.github.com/sintaxi/670654550d4b30354614c7cfb72d5bf5
https://twitter.com/surge_sh/status/986960078078013441

Check the status of the name server with `dig` or` nslookup`.

For example: I'm in Japan, so I temporarily referred to `sgp.surge.sh`.

**Comment by fu-sen** [comment]
Google Public DNS (`8.8.8.8` and `8.8.4.4`) returns a response, which looks like a cache.
It does not return with other DNS.

**Comment by fu-sen** [comment]
DigitalOcean isn't having any issues, so it looks like it's having issues at the Surge level.
https://status.digitalocean.com/

**Comment by sintaxi** [comment]
Thanks for reporting. We made some DNS adjustments about an hour ago. Likely related. Will revert the changes and then find the source of the issue.

**Comment by fu-sen** [comment]
I have confirmed that Surge's service has been restored. Thank you! 😄

**Reply by sintaxi** [reply]
Thanks for confirming!

**Comment by fu-sen** [comment]
I noticed a change. Apparently you applied this:
https://github.com/sintaxi/surge/discussions/444#discussioncomment-744204
